### PR TITLE
mark 1.14.3 as release

### PR DESCRIPTION
**Pull request by patricklodder**
Set the release marker. Draft until everything else is in.

### CODE REVIEW

**Review by rnicoll — APPROVED** [review]

**Review by langerhans — APPROVED** [review]

**Review by RevStevieD — COMMENTED** [review]
It has the opportunity to become a strong cryptocurrency￼

**Comment by kitcateleven** [comment]
Indeed

File: configure.ac [hunk 1]
```diff
@@ -4,7 +4,7 @@ define(_CLIENT_VERSION_MAJOR, 1)
 define(_CLIENT_VERSION_MINOR, 14)
 define(_CLIENT_VERSION_REVISION, 3)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
+define(_CLIENT_VERSION_IS_RELEASE, true)
```
[the remark this answers is not in the document]
Comment (Paddy613) [reply configure.ac:7]: Hello world

File: configure.ac [hunk 2]
```diff
@@ -4,7 +4,7 @@ define(_CLIENT_VERSION_MAJOR, 1)
...
 define(_COPYRIGHT_YEAR, 2021)
```
Comment (Paddy613) [inline configure.ac:8]: configure.ac

File: configure.ac [hunk 3]
```diff
@@ -4,7 +4,7 @@ define(_CLIENT_VERSION_MAJOR, 1)
...
 define(_COPYRIGHT_HOLDERS,[The %s developers])
```
[the remark this answers is not in the document]
Comment (Paddy613) [reply configure.ac:9]: ****

**Review by Paddy613 — COMMENTED** [review]
****

**Pull request merged**